web/rootfs/etc/cont-init.d/10-config

```diff
@@ -10,24 +10,30 @@ mkdir -p \
 # generate keys (maybe)
 if [[ $DISABLE_HTTPS -ne 1 ]]; then
     if [[ $ENABLE_LETSENCRYPT -eq 1 ]]; then
+        ACME_VERSION=3.0.6
+        validation=${LETSENCRYPT_VALIDATION:-"--standalone"}
+        echo "Using validation method: $validation"
         mkdir -p /config/acme.sh
-        pushd /opt
-        sh ./acme.sh --install --home /config/acme.sh --accountemail $LETSENCRYPT_EMAIL
-        popd
+        cd /tmp/
+        wget https://github.com/acmesh-official/acme.sh/archive/refs/tags/$ACME_VERSION.tar.gz
```

Comment:
Can you please write this more succinctly? wget -o/tmp/... | tar so we don't need to move / rename folders.